tox.ini

```diff
@@ -0,0 +1,13 @@
+[flake8]
+count = True
+#default_ignorelist: E121,E123,E126,E226,E24,E704,W503,W504
+ignore = U101
+	# temporarily disable docstring related stuff
+	D100, D101, D102, D103, D107, D204, D205, D400, D401
+inline-quotes = double
+
+per-file-ignores =
+	main.py: F401
+	# As long as numpy should be available in its current state
+	add_equation.py: F403, F401
+	transform_data.py: F403, F401
```

Comment:
I like this ability to do per-file-ignores. There's a sensible reason for using star imports here (so that a user can just type sin(X) instead of np.sin(X)), but I don't want to ignore the rule all together as in general theses kind of blank imports are a bad idea. (Hence these functionalities being isolated in their own file)